doc/getting-started/apl-nuc.rst

```diff
@@ -364,6 +364,76 @@ For hugeTLB to work, you'll need to reserve huge pages:
      by using a 2M page, but make sure your huge page reservation size is
      large enough for your usage.
 
+
+USB Device Sharing
+==========================================
+
+The ACRN hypervisor supports USB device sharing.  Suppose you have
+two keyboards and mice connected to your device, one keyboard and 
+mouse set for the SOS, and the other set for the UOS.  
+
```

Comment:
Since we have no other documentation on USB for now, we should add a note to indicate that the way USB device sharing is done in ACRN is by assigning a specific physical port (USB connector) to the Service OS or a Virtual Machine (UOS), this is true regardless of what device is attached to that physical port (this is different than other virtualization solutions which look for specific USB device and pass them to a VM, e.g. VMWare, Xen).

An important note too is that different `bus-port` combinations can be pointing at the same USB connector, this is typically seen for USB3.0 connectors which have both a USB2.0 controller (with its `bus-port` ID) and a USB3.0 controller (with another `bus-port`) attached to it (and the controller effectively used is the one matching the device plugged in, i.e. USB 2.0 or 3.0 device).

@hwang37 , could you assign someone to help write a more complete guide about USB device sharing?